cmd/sg.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+	"github.com/surajincloud/awsctl/pkg/ec2"
+)
+
+var sgCmd = &cobra.Command{
+
+	Use:   "sg",
```

Comment:
I think we should have `securitygroups, `securitygroup` and `sg` all the options here.

Reply:
You mean `awsctl get securitygroups` ??

Reply:
yes